app/serializers/listable_topic_serializer.rb

```diff
@@ -121,6 +122,16 @@ def unpinned
     PinnedCheck.unpinned?(object, object.user_data)
   end
 
+  def read_by_group_member
+    return false unless object.respond_to?(:minimum_unread_count)
```

Comment:
In what case would the object not have this method?

Reply:
I Should have added a comment here. This attribute is dynamically calculated [here](https://github.com/discourse/discourse/pull/7989/files?file-filters%5B%5D=.rb#diff-2c209e01ac5c122f000094f5c285e082R1063).

Reply:
I see, that should be OK although traditionally `respond_to?` is slow. We probably won't notice it.